packages/block-editor/src/components/dimensions-tool/aspect-ratio-tool.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	SelectControl,
+	__experimentalToolsPanelItem as ToolsPanelItem,
+} from '@wordpress/components';
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * @typedef {import('@wordpress/components/build-types/select-control/types').SelectControlProps} SelectControlProps
+ */
+
+/**
+ * These should use the same values as AspectRatioDropdown in @wordpress/block-editor.
```

Comment:
It would be good to add to the comment why the duplication is needed and can't be dried.

Reply:
It isn't that they can't be DRY, but this PR is already big enough and I'd have to update the post featured image block too. They also technically aren't the same right now either, so maybe a TODO comment that gets committed is more appropriate.